modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/JobEndpoint.java

```diff
@@ -424,6 +424,55 @@ public JObject getTasksAsJSON(WorkflowQuery query) throws JobEndpointException,
     return json;
   }
 
+  /**
+   * Returns the list of tasks matching the given query as JSON Object
+   *
+   * @param query
+   *          The worklfow query
+   * @return The list of matching tasks as JSON Object
+   * @throws JobEndpointException
+   * @throws NotFoundException
+   */
+  public JObject getTasksAsJSON(String mediapackageId) throws JobEndpointException, NotFoundException {
+    // Get results
+    WorkflowSet workflowInstances = null;
+    long totalWithoutFilters = 0;
+    List<JValue> jsonList = new ArrayList<>();
+
+    try {
+      workflowInstances = workflowService.getWorkflowInstances(mediapackageId);
+      totalWithoutFilters = workflowService.countWorkflowInstances();
+    } catch (WorkflowDatabaseException e) {
+      throw new JobEndpointException(String.format("Not able to get the list of job from the database: %s", e),
+              e.getCause());
+    }
+
+    WorkflowInstance[] items = workflowInstances.getItems();
+
+    for (WorkflowInstance instance : items) {
+      long instanceId = instance.getId();
+      String series = instance.getMediaPackage().getSeriesTitle();
```

Comment:
I'm again reiterating my request to use the term mpId versus "mediapackage"  to represent the mediapackage id in this pull :) . Traditionally, calls like "getMediaPackage", shown above, return the media package, versus just the id of the package. The addition of the term "mediapackage" in this pull, to represent the mediapackage identifier, not the mediapackage and adds confusion with the traditional use of the term "mediapackage" to represent the media package.

Reply:
+1